tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -1112,6 +1112,17 @@ Step #1 might already solve the problem, however if it still persists, execute s
 This issue occurs with new Anaconda installations when `pip` tries to remove `easy-install.pth`. 
 This file is not included in Anaconda packages, which causes the `pip` installation to fail.
 
+#### Cupti_wrapper.cc: Could not find cuptiActivityRegisterCallbacksin libcupti DSO
+
+If running TensorFlow Python script you encounter the error:
+```
+c:\tf_jenkins\home\workspace\nightly-win\device\gpu\os\windows\tensorflow\core\platform\default\gpu\cupti_wrapper.cc:59] Check failed: ::tensorflow::Status::OK() == (::tensor     flow::Env::Default()->GetSymbolFromLibrary( GetDsoHandle(), kName, &f  )) (OK vs. Not found: cuptiActivityRegisterCallbacks not found)could not find cuptiActivityRegisterCallb     acksin libcupti DSO
```

Comment:
Slightly weird spacing here? "tensor&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;flow" and "cuptiActivityRegisterCallb&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;acksin"